torchok/models/necks/segmentation/hrnet.py

```diff
@@ -30,4 +38,5 @@ def forward(self, features: List[Tensor]) -> List[Tensor]:
         x3 = F.interpolate(x3, size=(x0_h, x0_w), mode='bilinear', align_corners=True)
```

Comment:
It will fall in ONNX converter due to `align_corners=True`. Add comment `ONNX doesn't support align_corners=True`